src/MyTested.AspNetCore.Mvc.TempData/ComponentBuilderTempDataWithoutExtensions.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace MyTested.AspNetCore.Mvc
+{
+    using System;
+    using System.Collections.Generic;
+    using Builders.Contracts.Base;
+    using Builders.Contracts.Data;
+    using Builders.Base;
+    using Builders.Data;
+    using Internal.TestContexts;
+
+    /// <summary>
+    /// Contains <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.TempDataDictionary"/> extension methods for <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/>.
+    /// </summary>
+    public static class ComponentBuilderTempDataWithoutExtensions
+    {
+        /// <summary>
+        /// Remove entity by providing its key from <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.TempDataDictionary"/>.
+        /// </summary>
+        /// <typeparam name="TBuilder">Class representing ASP.NET Core MVC test builder.</typeparam>
+        /// <param name="builder">Instance of <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/> type.</param>
+        /// <param name="key">The key of the entity to be deleted.</param>
+        /// <returns>The same component builder.</returns>
+        public static TBuilder WithoutTempData<TBuilder>(
+            this IBaseTestBuilderWithComponentBuilder<TBuilder> builder,
+            string key)
+            where TBuilder : IBaseTestBuilder
+            => builder.WithoutTempData(tempData => tempData.WithoutEntry(key));
```

Comment:
Multiple chains. Put the on multiple lines like the example I gave earlier in the review.